int/int_test.go

```diff
@@ -0,0 +1,167 @@
+package int_test
+
+import (
+	"context"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	configv1 "github.com/openshift/api/config/v1"
+	"github.com/openshift/route-monitor-operator/api/v1alpha1"
+	. "github.com/openshift/route-monitor-operator/int"
+	"github.com/openshift/route-monitor-operator/pkg/util/templates"
+	monitoringv1 "github.com/prometheus-operator/prometheus-operator/pkg/apis/monitoring/v1"
+	"k8s.io/apimachinery/pkg/api/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/types"
+)
+
+var _ = Describe("Integrationtests", func() {
+	var (
+		i *Integration
+	)
+	BeforeSuite(func() {
+		var err error
+		i, err = NewIntegration()
+		Expect(err).NotTo(HaveOccurred())
+	})
+	AfterSuite(func() {
+		i.Shutdown()
+	})
+
+	Context("ClusterUrlMonitor creation", func() {
+		var (
+			clusterUrlMonitorName      string
+			clusterUrlMonitorNamespace string
+			clusterUrlMonitor          v1alpha1.ClusterUrlMonitor
+			expectedServiceMonitorName types.NamespacedName
+		)
+		BeforeEach(func() {
+			err := i.RemoveClusterUrlMonitor(clusterUrlMonitorNamespace, clusterUrlMonitorName)
+			clusterUrlMonitorName = "fake-url-monitor"
+			clusterUrlMonitorNamespace = "default"
+			expectedServiceMonitorName = templates.TemplateForServiceMonitorName(clusterUrlMonitorNamespace, clusterUrlMonitorName)
+			Expect(err).NotTo(HaveOccurred())
```

Comment:
shouldn't this error be much higher, as the `TemplateForServiceMonitorName` doesn't return that error?

Reply:
done